pkg/secrets/vault.go

```diff
@@ -55,6 +56,22 @@ func (c *vaultClient) getSecretAtPath(path, key string) ([]byte, error) {
 	return ret, err
 }
 
+func (c *vaultClient) setItemAtPath(path, field string, content string) error {
+	path = c.pathFor(path)
+	var data map[string]string
+	if current, err := c.upstream.GetKV(path); err != nil {
+		if !vaultclient.IsNotFound(err) {
+			return err
+		}
+		data = map[string]string{field: content}
+	} else {
+		data = current.Data
+		data[field] = content
+	}
```

Comment:
I forgot to mention: this is ugly, but required to conform to the current client interface based on the MO of the Bitwarden client.

I decided not to do anything too clever until we have some measurement of the overhead (or remove the old Bitwarden client so a more fitting interface can be used).